.prettierignore

```diff
@@ -0,0 +1,19 @@
+# Ignore everything except as negated below
+*
+
+!*.scss
+!*.md
+!*.yml
+!*.json
+
+# Ensures subdirectories are checked for files matching the above
+!*/
+
+config/locales/
+app/views/content/data_sharing_agreement.md
```

Comment:
Prettier tried to make some dubious changes to this file, possibly because it's currently actually a bit off. Given it's fairly important that it stays static I think it makes sense to ignore it.